helm/values.yaml

```diff
@@ -368,10 +379,11 @@ serverConfig:
 initBackfillJob:
   # -- By default, all image settings use those set in insightsController.server. Optionally use the below to override. This should not be common.
   # imagePullSecrets: []
-  # image:
-  #   repository: ghcr.io/cloudzero/cloudzero-insights-controller/cloudzero-insights-controller
-  #   tag: 0.1.1
-  #   pullPolicy: Always
+  image:
+    repository:
+    tag:
+    revision:
+    pullPolicy:
```

Comment:
is it more idiomatic to comment these out, or leave them empty? I don't have a preference either way

Reply:
sounds good to me!